src/js/components/common/SelectWithLabel.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import styles from './SelectWithLabel.module.css';
+import cx from 'classnames';
+import Error from './Error';
+import { InputHTMLAttributes } from 'react';
+
+type ErrorType = {
+  message?: string | undefined;
+};
+
+export type SelectOptions = {
+  placeholder: string;
+  options: Array<string>;
+};
+
+export type SelectWithLabelProps = InputHTMLAttributes<HTMLSelectElement> & {
+  label: string;
+  error?: ErrorType;
+  selectProps: SelectOptions;
```

Comment:
zamiast selectProps, rozpisałbym je tutaj, ewentualnie 
```suggestion
  export type SelectWithLabelProps = InputHTMLAttributes<HTMLSelectElement> & {
  label: string;
  error?: ErrorType;
  ...SelectOptions;
```